pkg/health/health.go

```diff
@@ -0,0 +1,161 @@
+	cephHealthDetails := exec.RunCommandInOperatorPod(context, "ceph", []string{"health", "detail"}, operatorNamespace, clusterNamespace, false)
+	if strings.Contains(cephHealthDetails, "HEALTH_OK") {
+		fmt.Printf("\n%s", cephHealthDetails)
+	} else if strings.Contains(cephHealthDetails, "HEALTH_WARN") {
+		fmt.Println("Warning: ", cephHealthDetails)
+	} else if strings.Contains(cephHealthDetails, "HEALTH_ERR") {
+		fmt.Println("Error: ", cephHealthDetails)
+	}
+}
+
+func CheckAllPodsStatus(context *k8sutil.Context, operatorNamespace, clusterNamespace string) {
+
+	var podNotRunning, podRunning []v1.Pod
+	podRunning, podNotRunning = getPodRunningStatus(context, operatorNamespace)
+	if operatorNamespace != clusterNamespace {
+		clusterRunningPod, clusterNotRunningPod := getPodRunningStatus(context, clusterNamespace)
+		podRunning = append(podRunning, clusterRunningPod...)
+		podNotRunning = append(podNotRunning, clusterNotRunningPod...)
+	}
+
+	fmt.Println("Info: Pods that are in 'Running' status")
+	for i := range podRunning {
+		fmt.Println(podRunning[i].Name, "\t", podRunning[i].Status.Phase, "\t", podRunning[i].Namespace, "\t", podRunning[i].Spec.NodeName)
+	}
+
+	fmt.Println("\nWarning: Pods that are 'Not' in 'Running' status")
+	for i := range podNotRunning {
+		fmt.Println(podNotRunning[i].Name, "\t", podNotRunning[i].Status.Phase, "\t", podNotRunning[i].Namespace, "\t", podNotRunning[i].Spec.NodeName)
+	}
+}
+
+func getPodRunningStatus(context *k8sutil.Context, namespace string) ([]v1.Pod, []v1.Pod) {
+	var podNotRunning, podRunning []v1.Pod
+	podList, err := context.Clientset.CoreV1().Pods(namespace).List(ctx.TODO(), metav1.ListOptions{})
+	if err != nil {
+		fmt.Printf("\nfailed to list pods in namespace %s: %v\n", namespace, err)
+	}
+
+	for i := range podList.Items {
+		if podList.Items[i].Status.Phase != v1.PodRunning {
+			podNotRunning = append(podNotRunning, podList.Items[i])
+		} else {
+			podRunning = append(podRunning, podList.Items[i])
+		}
+	}
+	return podRunning, podNotRunning
+}
+
+func checkPgStatus(context *k8sutil.Context, operatorNamespace, clusterNamespace string) {
+	pg_stat_code := exec.RunCommandInOperatorPod(context, "ceph", []string{"pg", "stat"}, operatorNamespace, clusterNamespace, false)
+
+	if strings.Contains(pg_stat_code, "down") || strings.Contains(pg_stat_code, "incomplete") || strings.Contains(pg_stat_code, "snaptrim_error") {
+		fmt.Printf("\nError: %s\n", pg_stat_code)
+	} else if strings.Contains(pg_stat_code, "active+clean;") {
+		fmt.Printf("\nInfo: %s\n", pg_stat_code)
+	} else {
+		fmt.Printf("\nWarning: %s\n", pg_stat_code)
+	}
+}
+
+func checkMgrPodsStatusAndCounts(context *k8sutil.Context, operatorNamespace, clusterNamespace string) {
+	opts := metav1.ListOptions{LabelSelector: "app=rook-ceph-mgr"}
+	podList, err := context.Clientset.CoreV1().Pods(clusterNamespace).List(ctx.TODO(), opts)
+	if err != nil {
+		fmt.Printf("\nfailed to list mgr pods with label %s: %v\n", opts.LabelSelector, err)
```

Comment:
need to return here? the `podList` won't be valid below
```suggestion
		fmt.Printf("\nfailed to list mgr pods with label %s: %v\n", opts.LabelSelector, err)
		return
```